js/app.js

```diff
@@ -1 +1,80 @@
-// Your code here
+// Model
+
+$(document).ready(function() {
+
+var model = {
+      loggedIn: false,
+      userId: '',
+      username: '',
+      category: [],
+      articles: []
+  };
+
+// View
+
+function renderUser() {
+  var templateSource = $('#user-template').html();
+  var template = Handlebars.compile(templateSource);
+  var templateRender = template(model);
+  $('#loginContainer').html(templateRender);
+};
+
+renderUser();
+
+// Controller
+// event listners and ajax
+
+$('#loginSubmit').on('click', addUser);
+
+function addUser() {
+  var user = $('#username').val();
+  $.ajax {
+    url: '/login',
+    type: POST,
+    data: { username: username },
+  }
+}
+
+function renderArticles() {
+  var templateSource = $('#article-template').html();
+  var template = Handlebars.compile(templateSource);
+  var templateRender = template(model);
+  $('#main').html(templateRender);
+};
+
+renderArticles();
+
+function getArticles() {
+
+  $.get('/feed', function(data) {  
+    model.article.id = data.id; 
```

Comment:
Here, looking at the api docs, we'd notice that `data` is an array of article objects. Thus, we couldn't access `data.id`, but we could probably access `data[0].id`, etc. Instead of breaking out each property of each article, it might be easier to just save the whole data array directly in the model:
```js
model.articles = data
```